railties/test/isolation/abstract_unit.rb

```diff
@@ -478,6 +483,7 @@ def use_frameworks(arr)
 
       if to_remove.include?(:activerecord)
         remove_from_config "config.active_record.*"
+        remove_from_config ".*config.active_record.*\n"
```

Comment:
Same thing as before, any reason to not make these a single line?

Reply:
Good call, fixed in 7d74aedee9 and same as above